runelite-client/src/main/java/net/runelite/client/plugins/kourendlibrary/KourendLibraryOverlay.java

```diff
@@ -236,6 +243,20 @@ public Dimension render(Graphics2D g)
 		return null;
 	}
 
+	private boolean isDarkManuscript(@Nullable Book book)
```

Comment:
Create some set of the dark manuscripts and use `DARK_MANUSCRIPTS.includes(book)` in place of method.

Reply:
Okay, fresh set of eyes on it, I realized that there was already an existing boolean in the Book class that flagged if it was a Dark Manuscript or not, and that is set at instantiation by the constructor.  With that in mind, I removed my method and just called the existing public getter for that boolean.  I can still implement an EnumSet in the Overlay class to check against if you want, but I think it's cleaner like this.